config_schema.json

```diff
@@ -170,6 +170,15 @@
           "description": "The port to use for the Zeroconf service (empty for random)",
           "default": 0
         },
+        "zeroconf_backend": {
```

Comment:
Perhaps add something to the README to explain the two backens and why to use one over the other?